app/assets/stylesheets/parameters.scss

```diff
@@ -20,3 +20,13 @@ $table-border-factor: .1;
 $list-group-hover-bg: rgba(var(--bs-emphasis-color-rgb), .075);
 
 $enable-negative-margins: true;
+
+.role-icon-administrator {
+  color: #f69e42;
+}
+.role-icon-moderator {
+  color: #447eff;
+}
+.role-icon-importer {
+  color: #38e13a;
+}
```

Comment:
These aren't really parameters. are they?

Reply:
Put them in another file, `colors.scss`.

I hope to use `common.scss` for print style later, Bootstrap has [some support for that](https://getbootstrap.com/docs/5.3/utilities/display/#display-in-print).